src/fullscreen/fullscreen-controller.js

```diff
@@ -204,8 +206,7 @@ class FullscreenController {
     Promise.resolve(this._nativeExitFullScreen()).then(
       () => {
         this._isInFullscreen = false;
-        const screenOrientation = screen && screen.orientation && typeof screen.orientation.lock === 'function';
-        if (screenOrientation) {
+        if (this._isScreenOrientationSupport) {
```

Comment:
We should check if we locked it before unlocking as there could be a case where app locked it and we shouldn't change that.

Reply:
If you set any as default it would be redundant to check if I locked before we'll always make the lock...
so I changed it back to an invalid value to avoid this lock by default.